imports/plugins/core/orders/client/templates/workflow/shippingTracking.js

```diff
@@ -115,7 +123,7 @@ Template.coreOrderShippingTracking.helpers({
     const currentData = Template.currentData();
     const order = Template.instance().order;
 
-    const shippedItems = _.every(currentData.fulfillment.items, (shipmentItem) => {
+    const shippedItems = _.every(currentData.fulfillment && currentData.fulfillment.items, (shipmentItem) => {
```

Comment:
This should use prototype `every` instead of `lodash` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every